src/wallet/wallettool.cpp

```diff
@@ -103,25 +103,27 @@ static void WalletShowInfo(CWallet* wallet_instance)
     tfm::format(std::cout, "Address Book: %zu\n", wallet_instance->m_address_book.size());
 }
 
-bool ExecuteWalletToolFunc(const std::string& command, const std::string& name)
+bool ExecuteWalletToolFunc(const ArgsManager& args, const std::string& command, const std::string& name)
 {
     fs::path path = fs::absolute(name, GetWalletDir());
 
-    // -format is only allowed with createfromdump. Disallow it for all other commands.
-    if (gArgs.IsArgSet("-format") && command != "createfromdump") {
+    if (args.IsArgSet("-format") && command != "createfromdump") {
         tfm::format(std::cerr, "The -format option can only be used with the \"createfromdump\" command.\n");
         return false;
     }
-    // -dumpfile is only allowed with dump and createfromdump. Disallow it for all other commands.
-    if (gArgs.IsArgSet("-dumpfile") && command != "dump" && command != "createfromdump") {
+    if (args.IsArgSet("-dumpfile") && command != "dump" && command != "createfromdump") {
         tfm::format(std::cerr, "The -dumpfile option can only be used with the \"dump\" and \"createfromdump\" commands.\n");
         return false;
     }
+    if (args.IsArgSet("-descriptors") && command != "create") {
+        tfm::format(std::cerr, "The -descriptors option can only be used with the 'create' command.\n");
```

Comment:
the other error messages use double quotes around the command names (lines 112, 117)
```suggestion
        tfm::format(std::cerr, "The -descriptors option can only be used with the \"create\" command.\n");
```

Reply:
the help text uses single quotes

Reply:
yup, i noticed the help follows a different format than the errors